bench/bench.py

```diff
@@ -262,7 +260,6 @@ def sync(
 	):
 		if required == UNSET_ARG:
 			required = []
-		self.initialize_apps()
```

Comment:
This was the source of randomness in ordering. `self.apps` is already present and "updated" with whatever changes are made to it.